controllers/controller.js

```diff
@@ -34,3 +36,16 @@ exports.fetchArticlebyId = (req, res, next) => {
       next(err);
     });
 };
+
+exports.fetchCommentsByArticleId = (req, res, next) => {
+  const { article_id } = req.params;
+  const promise1 = fetchCommentsByIdFromDB(article_id);
```

Comment:
I dont know promise1 and promise2 are the best variable names here, you could be a little more descriptive